src/odin/narrativebuilder.cc

```diff
@@ -3809,6 +3858,393 @@ std::string NarrativeBuilder::FormVerbalPostTransitionTransitInstruction(Maneuve
   return instruction;
 }
 
+std::string NarrativeBuilder::FormVerbalSuccinctStartTransitionInstruction(Maneuver& maneuver) {
+  // "0": "Head <CARDINAL_DIRECTION>.",
+  // "1": "Head <CARDINAL_DIRECTION> for <LENGTH>.",
+  // "5": "Drive <CARDINAL_DIRECTION>.",
+  // "6": "Drive <CARDINAL_DIRECTION> for <LENGTH>.",
+  // "10": "Walk <CARDINAL_DIRECTION>.",
+  // "11": "Walk <CARDINAL_DIRECTION> for <LENGTH>.",
+  // "15": "Bike <CARDINAL_DIRECTION>."
+  // "16": "Bike <CARDINAL_DIRECTION> for <LENGTH>.",
+
+  std::string instruction;
+  instruction.reserve(kInstructionInitialCapacity);
+
+  // Set cardinal_direction value
+  std::string cardinal_direction =
+      dictionary_.start_verbal_subset.cardinal_directions.at(maneuver.begin_cardinal_direction());
+
+  // Determine which phrase to use
+  uint8_t phrase_id = 0;
+
+  // Set base phrase id per mode
+  if (maneuver.travel_mode() == TripLeg_TravelMode_kDrive) {
+    phrase_id += 5;
+  } else if (maneuver.travel_mode() == TripLeg_TravelMode_kPedestrian) {
+    phrase_id += 10;
+  } else if (maneuver.travel_mode() == TripLeg_TravelMode_kBicycle) {
+    phrase_id += 15;
+  }
+
+  if (maneuver.include_verbal_pre_transition_length()) {
+    // Increment phrase id for length
+    phrase_id += 1;
+  }
+
+  // Set instruction to the determined tagged phrase
+  instruction = dictionary_.start_verbal_subset.phrases.at(std::to_string(phrase_id));
+
+  // Replace phrase tags with values
+  boost::replace_all(instruction, kCardinalDirectionTag, cardinal_direction);
+  boost::replace_all(instruction, kLengthTag,
+                     FormLength(maneuver, dictionary_.start_verbal_subset.metric_lengths,
+                                dictionary_.start_verbal_subset.us_customary_lengths));
+
+  // If enabled, form articulated prepositions
+  if (articulated_preposition_enabled_) {
+    FormArticulatedPrepositions(instruction);
+  }
+
+  return instruction;
+}
+
+std::string NarrativeBuilder::FormVerbalSuccinctDestinationTransitionInstruction() {
```

Comment:
remove for now

Reply:
I'm going to leave for now since we discussed this.